cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -102,20 +131,49 @@
                 vertical-align: bottom;
             }
         } );
+
+        &_left {
+            .u-webfont-medium();
+            display: inline-block;
+            float: none;
+            font-size: 1.125em;
+            grid-area: category;
+            line-height: 1.33333;
+
+            .m-meta-header_separator {
+                padding-left: 0.25em;
+                padding-right: 0.25em;
+            }
+        }
+
+        &_right {
+            display: inline-block;
+            float: none;
```

Comment:
Possibly the shared settings could be combined into:

```less
&_left,
&_right {
    display: inline-block;
    float: none;
}
```

Reply:
If that's preferable, sure!